src/app/public/modules/inline-form/inline-form.component.ts

```diff
@@ -0,0 +1,236 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  OnInit,
+  OnDestroy,
+  ElementRef,
+  TemplateRef
+} from '@angular/core';
+
+import {
+  Observable
+} from 'rxjs/Observable';
+
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
+import 'rxjs/add/observable/zip';
+
+import {
+  SkyLibResourcesService
+} from '@skyux/i18n';
```

Comment:
To make the style rule more simple, I think we should list all "third-party" modules in alphabetical order (which would include `@skyux` stuff). We don't give preference to Blackbaud stuff in our style guide. What do you think?
https://docs.blackbaud.com/skyux-team-docs/style#imports